playbooks/install_stack.yaml

```diff
@@ -62,6 +62,11 @@
       src: containers-prepare-parameters.yaml.j2
       dest: "{{ ansible_env.HOME }}/containers-prepare-parameters.yaml"
 
+  - name: Create StandaloneSriov Role
+    shell: |
+      cp /usr/share/openstack-tripleo-heat-templates/roles/Standalone.yaml {{ ansible_env.HOME }}/Standalone.yaml
+      echo "    - OS::TripleO::Services::NeutronSriovAgent" >> $HOME/Standalone.yaml
+
```

Comment:
This is used to create the Standalone SRIOV role. 
Tried using StandaloneSRIOV.yaml as the name but there were too many places where Standalone would need to be used so just left as same name. Probably can be named better with more investigation

Reply:
For this one, I'll see if we can add OS::TripleO::Services::NeutronSriovAgent into the upstream Standalone role. It's disabled by default anyway:

```
overcloud-resource-registry-puppet.j2.yaml:
166:  OS::TripleO::Services::NeutronSriovAgent: OS::Heat::None
```